boards/shields/xenvm_dom0/Kconfig.defconfig

```diff
@@ -6,4 +6,9 @@ if SHIELD_XENVM_DOM0
 config HEAP_MEM_POOL_SIZE
 	default 2097152
 
+# it is needed at least 16M for grant tables
+# 32M should be enough at the beginning
```

Comment:
It is wrong to use "at the beginning" in this context. Ukrainian common phrases does not translate directly into English ones.

You can write "should be enough for basic use-cases".